x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/job_details_step/components/advanced_section/advanced_section.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiAccordion, EuiSpacer } from '@elastic/eui';
+import { ModelPlotSwitch } from './components/model_plot';
+import { DedicatedIndexSwitch } from './components/dedicated_index';
+import { ModelMemoryLimitInput } from './components/model_memory_limit';
+
+const ButtonContent = <div>Advanced</div>;
```

Comment:
Just curious why this needs to be wrapped in a div?

Reply:
it was to add styling, it looks a bit plain IMO, but i'm still not sure what it should look like. i'll remove the div for now